openpype/hosts/maya/api/shader_definition_editor.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+"""Editor for shader definitions.
+
+Shader names are stored as simple text file over GridFS in mongodb.
+
+"""
+import os
+from Qt import QtWidgets, QtCore, QtGui
+from openpype.lib.mongo import OpenPypeMongoConnection
+from openpype import resources
+import gridfs
+
+
+class ShaderDefinitionsEditor(QtWidgets.QWidget):
+    """Widget serving as simple editor for shader name definitions."""
+
+    # name of the file used to store definitions
+    DEFINITION_FILENAME = "maya/shader_definition.txt"
```

Comment:
I would preffer to have this vaaible defined at one place (as it's used here and in validator).

Reply:
I agree, but I have no idea where as this file can be used elsewhere - for example in other tool scripts doing renaming stuff, etc. Probably under *Projects > Maya > Shader name definition file*?

Reply:
Maybe not in settings but in code? It file in database (as I understood) so why to have it modifiable for studio?